rust/cli/output

```diff
@@ -100,9 +100,9 @@ test.sh: Shell script (code) 99%
   }
 ]
 + magika test.sh --jsonl
-{"path":"test.sh","result":{"status":"ok","value":{"dl":{"description":"Shell script","extensions":["sh"],"group":"code","is_text":true,"label":"shell","mime_type":"text/x-shellscript"},"output":{"description":"Shell script","extensions":["sh"],"group":"code","is_text":true,"label":"shell","mime_type":"text/x-shellscript"},"score":0.9980000257492065}}}
+{"path":"test.sh","result":{"status":"ok","value":{"dl":{"description":"Shell script","extensions":["sh"],"group":"code","is_text":true,"label":"shell","mime_type":"text/x-shellscript"},"output":{"description":"Shell script","extensions":["sh"],"group":"code","is_text":true,"label":"shell","mime_type":"text/x-shellscript"},"score":0.9990000128746033}}}
```

Comment:
did we ever figure out the source of this non-determinism?

Reply:
This is not non-determinism. I'm modifying the file (it's recursive, I use the test script itself as input). I could also use a file from the basic test suite, probably better.

Reply:
Ok, I'm using `basic/rust/code.rs` and `basic/python/code.py` which hopefully shouldn't change often.